components/mpas-ocean/cime_config/config_compsets.xml

```diff
@@ -142,6 +142,11 @@
     <lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI%DIB_MPASO%IBDISMFDATMFORCED_DROF%JRA-1p4-2018-AIS0ROF_SGLC_SWAV</lname>
   </compset>
 
+  <compset>
+    <alias>GMPAS-JRA1p4-DIB-PISMF-DIS</alias>
+    <lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI%DIB_MPASO%IBPISMFDATMFORCED_DROF%JRA-1p4-2018-AIS0ROF_MALI%DATA_SWAV</lname>
```

Comment:
based on discussion, change to `JRA-1p4-2018-AIS0ICE` option?

Reply:
and switch from 1p4 to 1p5?

Reply:
Based on Slack discussion, wait for a PR that fixes things so runoff is removed from G-Cases as it is from B-Cases.  The, yes, switch to JRA1p5 but do not use any of the `-AIS0*` options so that MPASO takes care of the zeroing out.

Reply:
Update note: this PR will be rebased, and compset adjustments made, once https://github.com/E3SM-Project/E3SM/pull/6693 is merged.

Reply:
We need not to forget to do the rebase once https://github.com/E3SM-Project/E3SM/pull/6693 goes in, which I think will be soon.